lib/osf-components/addon/components/tags-widget/template.hbs

```diff
@@ -1,7 +1,7 @@
 {{#unless @taggable.tags.length}}
-    <span data-test-no-tags>
+    <div data-test-no-tags>
```

Comment:
I'm a little unsure if we want this change, as this is going to change the layout of this component wherever it's invoked. Might be more prudent to style the wrapper div in `app/preprints/-components/submit/metadata/template.hbs` to block-display instead.

Reply:
Ok, so I had a look through to see where else we use this component, and it looks like we should be fine to change this to a `div`